bin/ch/stdafx.h

```diff
@@ -63,7 +63,7 @@
 #if defined(DBG)
 
 #define _STRINGIZE_(x) #x
-#define _STRINGIZE(x) _STRINGIZE_(x)
+//#define _STRINGIZE(x) _STRINGIZE_(x)
```

Comment:
I don't think this should be removed. What if the parameter is a macro? [http://stackoverflow.com/questions/8231966/why-do-i-need-double-layer-of-indirection-for-macros](http://stackoverflow.com/questions/8231966/why-do-i-need-double-layer-of-indirection-for-macros)

---
In reply to: [70015278](https://github.com/Microsoft/ChakraCore/pull/1254#discussion_r70015278) [](ancestors = 70015278)